src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/wwwroot/css/site.css

```diff
@@ -46,19 +46,19 @@ a, .btn-link {
     z-index: 1000;
 }
 
-    #blazor-error-ui .dismiss {
```

Comment:
I think it's done this way on purpose because this is a sub-style of the style above it. Obviously it's just _styles_ (in a style??!?! haha!), but it's a fairly common pattern.

I have no personal opinion on this, but it was definitely deliberate.